### PR TITLE
UIBULKED-352: Localize numbers displayed in bulk edit

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
[UIBULKED-352](https://issues.folio.org/browse/UIBULKED-352)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/BulkEdit.test.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { runAxeTest } from '@folio/stripes-testing';
 
 import '../../test/jest/__mock__';
 
+import { IntlProvider } from 'react-intl';
```
Comment (BogdanDenis) [inline src/components/BulkEdit.test.js:13]: Please move the import higher to be with other third party imports

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/BulkEdit.test.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEdit.test.js]: done!

**Review by zburke — CHANGES_REQUESTED** [review]
Bad news/good news: much of this work was not done correctly, but it's actually easier than what you did here. You don't have to directly format the numeric values passed into your messages; you just have to mark up the values in translation files using [ICU Message Syntax](https://formatjs.io/docs/core-concepts/icu-syntax#number-type) so the parser knows it's handling a number, e.g. updating
```
  "list.logSubTitle.matched": "{count} records match",
```
in `translations/ui-bulk-edit/en.json` like
```
  "list.logSubTitle.matched": "{count, number} records match",
```

**Comment by UladzislauKutarkin** [comment]
> Bad news/good news: much of this work was not done correctly, but it's actually easier than what you did here. You don't have to directly format the numeric values passed into your messages; you just have to mark up the values in translation files using [ICU Message Syntax](https://formatjs.io/docs/core-concepts/icu-syntax#number-type) so the parser knows it's handling a number, e.g. updating
> 
> ```
>   "list.logSubTitle.matched": "{count} records match",
> ```
> 
> in `translations/ui-bulk-edit/en.json` like
> 
> ```
>   "list.logSubTitle.matched": "{count, number} records match",
> ```

Thanks @zburke , I done it as you said. Looks more cleaner and easy to do in that way number formatting

File: translations/ui-bulk-edit/en.json [hunk 2]
```diff
@@ -61,8 +61,8 @@
   "list.preview.table.status.false": "Inactive",
 
   "list.errors.title": "Errors",
-  "list.errors.info": "<b>{fileName}: {entries} entries * {matched} records matched * {errors} errors</b>",
-  "list.errors.infoProcessed": "<b>{fileName}: {entries} entries * {matched} records changed * {errors} errors</b>",
+  "list.errors.info": "<b>{fileName}: {entries} entries * {matched, number} records matched * {errors, number} errors</b>",
+  "list.errors.infoProcessed": "<b>{fileName}: {entries} entries * {matched, number} records changed * {errors, number} errors</b>",
```
Comment (zburke) [inline translations/ui-bulk-edit/en.json:65]: `{entries}` needs the `{entries, number}` treatment too. There are also a few others (lines 246, 347, 386, 389) containing numeric substitutions. Your call on whether to fix 'em all or just those that are specifically in the scope of this story.

Comment (vashjs) [reply translations/ui-bulk-edit/en.json]: Hi @zburke, updated etries, and all poited lines.

**Review by BogdanDenis — APPROVED** [review]

**Review by zburke — APPROVED** [review]
This looks great now! Thanks for your persistence.

**Pull request merged**